src/DotNetWorker.Core/Hosting/WorkerMiddlewareWorkerApplicationBuilderExtensions.cs

```diff
@@ -75,15 +75,13 @@ public static IFunctionsWorkerApplicationBuilder UseOutputBindingsMiddleware(thi
                 return builder;
             }
 
-            builder.Services.AddSingleton<OutputBindingsMiddleware>();
+            builder.Services.AddSingleton<OutputBindingsMiddleware>(p => throw new InvalidOperationException($"Type {nameof(OutputBindingsMiddleware)} cannot be resolved using the container."));
```

Comment:
I understand that if this type is requested, it will thrown an exception now. But I am wondering, do we even need to make this call?

Reply:
Talked to Fabio, and this change makes sense. Leaving this registration call and the check right above prevents duplicate registration of this same middleware.

Reply:
Followed up offline.